index.html

```diff
@@ -3,9 +3,34 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <style>
+      body {
+        background-color: lightblue;
+        background-repeat: no-repeat;
+        background-position: center center;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+    </style>
   </head>
   <body>
     <div id="app"></div>
     <script src="/dist/build.js"></script>
+    <script>
+      (function() {
+        axios.get('https://api.unsplash.com/photos/random?featured&orientation=landscape', {
+          params: {
+            client_id: 'b325d6aa7cdaacc3b279aa500e0f091c1e3ae2b986270547a005dfe1835b78a9',
+            collections: 135648
+          }
+        }).then(function (response) {
+          // console.log(response);
+          document.body.style.backgroundImage = "url(" + response.data.urls.regular + ")";
+        }).catch(function (error) {
+          console.log(error);
```

Comment:
Over here we'd want a function that picks a random color in case Unsplash doesn't load. Let's source these hex codes into an array and pick random ones on reload. https://flatuicolors.com/ 

Just don't pick the light gray ones since our font color will be white, but I think the rest should work 👍